keyboards/ckeys/handwire_101/keymaps/default/keymap.c

```diff
@@ -0,0 +1,179 @@
+#include "handwire_101.h"
+#include "action_layer.h"
+
+#define _BASE           0  // base layer
+#define _LAYERS         1  // layer of all layers
+#define _MUSIC          2  // music mode
+#define _MUSIC_4_LIFE   3  // music mode until unplugged
+#define _MOUSE          4  // mousekeys
+#define _TERMINAL       5  // terminal
+#define _ADMIN          6  // admin duties
+
+#define _____ KC_TRNS
+
+enum custom_keycodes {
+  TERM_ABOUT = SAFE_RANGE,
+  TERM_PRINT,
+  TERM_FLUSH,
+  TERM_HELP,
+  CKEYS_ABOUT,
+};
+
+extern keymap_config_t keymap_config;
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* BASE (numpad)
+   * ,-----------------------.
+   * |  7  |  8  |  9  |  /  | <-- Hold for LAYERS
+   * |-----+-----+-----+-----|
+   * |  4  |  5  |  6  |  *  |
+   * |-----+-----+-----+-----|
+   * |  1  |  2  |  3  |  -  |
+   * |-----+-----+-----+-----|
+   * |  0  |  .  |  =  |  +  |
+   * `---------------------- '
+   */
+  [_BASE] = KEYMAP(
```

Comment:
All these instances of `KEYMAP` should be changed to `LAYOUT`.

```suggestion
  [_BASE] = LAYOUT(
```